proposal/package-importer.d.ts.md

```diff
@@ -330,6 +330,10 @@ Package Importers must reject the following patterns:
 * A URL whose path begins with `/`.
 * A URL with non-empty/null username, password, host, port, query, or fragment.
 
+If the conventions or specifications for an environment disallow any other URL
+patterns, the Package Importer must not reject, and instead return `null`. This
```

Comment:
"reject" isn't a well-defined term here. I'd say "must return `null` rather than throwing an error."

Reply:
Updated in https://github.com/sass/sass/pull/3777/commits/26d30d7b62448ee60074eb6e8d6a5cbbd727cb0d